src/main.tsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './main.css';
```

Comment:
Would it make sense to keep a `main.scss` or `globals.scss` to define global styles for the application such as the font-family? We don't set this anywhere now, right? For some reason the buttons seem to use a different font-family than the rest of the application, but I am not sure why:

<img width="250" alt="Screenshot 2024-03-14 at 09 18 28" src="https://github.com/christophblessing/quiz-recap/assets/62883011/4e5a99aa-4b53-4372-9ff3-3b2c90052472">

Reply:
Do you have a favourite `font-family`?

Reply:
We can go with a sans-serif system font stack, e.g:
```css
font-family: system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

I took it from [this article](https://css-tricks.com/snippets/css/system-font-stack/)